use_case_examples/deployment/server/deploy_to_aws.py

```diff
@@ -23,7 +23,7 @@
 
 import boto3
 
-from ..deployment.utils import filter_logs, wait_for_connection_to_be_available
+from .utils_server import filter_logs, wait_for_connection_to_be_available
```

Comment:
are you missing some files here? in the `utils_server` dir?

Reply:
utils_server is a py file. Should be good.